data/tabular/potassium_ion_channel_kir2_1_butkiewicz/transform.py

```diff
@@ -57,12 +57,12 @@ def get_and_transform_data():
                 "units": None,
                 "type": "boolean",
                 "names": [
-                    {"verb": "is blocking potassium ion channel activity"},
-                    {"verb": "blocks potassium ion channel activity"},
+                    {"gerund": "blocking of the potassium ion channel activity"},
```

Comment:
```suggestion
                    {"gerund": "blocking the potassium ion channel"},
```

Reply:
Maybe use:
block channel
decrease activity
but "block activity" sounds strange, but maybe this is used in the literature?